spec/order_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'order'
+
+describe Order do
+  it { is_expected.to respond_to :display_menu}
+
+  it { is_expected.to respond_to :add_to_order}
```

Comment:
You might need a more detailed test for this particular method, it passes but doesn't describe what it does or what you expect it to do.